src/main/kotlin/exercise3/task2/ParenthesesClusters.kt

```diff
@@ -24,7 +24,35 @@ package exercise3.task2
  */
 
 internal fun String.splitToBracketsClusters(): List<String> {
-    TODO("Implement me!!!")
+    val stack = mutableListOf<Char>()
```

Comment:
Your solution is correct in the task setup where only `(` and `)` brackets are allowed. 
Since I saw your implementation of the `exercises3.task1`, I think this was just an oversight from your side. :) 

In the task setup following requirement was set: 
``` * The following parenthesis pairs can appear in an expression string input: {}, [], ()```

Reply:
I see now that tests didn't cover the case for {} and [] brackets, so I guess it's fine. :)